docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -204,55 +248,93 @@ HotBackup resource name::
 include::ROOT:example$/hazelcast-persistence-restore-cr-name.yaml[]
 ----
 
-<1> `HotBackup` resource name used for the restore. The bucket URI and secret are taken from the `HotBackup` resource.
-<2> Agent responsible for restoring data from external storage.
-    The agent configuration is optional. If you provide `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
+<1> Name of the `HotBackup` resource used for the restore. The bucket URI and secret are taken from the `HotBackup` resource.
+<2> Optional. Agent responsible for restoring data from external storage.
+    If you provide `restore` under `persistence` and do not pass the agent configuration, Hazelcast Platform Operator
```

Comment:
same

Reply:
For future reference; if you have commented once, you can leave it at that; I'll ensure that all instances of the same information are updated in line.

Should save you a little time ;-)

Reply:
Updated